### PR TITLE
3109 webprofile10

### LINKED ISSUE #3109: Only show Web Profile 10 download package when possible
If Web Profile 10 is available for download from DHE, make that the only visible download package choice for Web Profile.

Filename for WP10 on DHE will be 
`openliberty-webProfile10-<release version>.zip`

### PR DESCRIPTION
**Pull request by kinueng**
## What was changed and why?

No regressions with the existing packages here with the code deployed here https://ui-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/start/#runtime_releases

As much of the code comments already document, this is a defensive coding approach to make sure the newest Web Profile version appears and the older Web Profile versions are hidden. Ideally the files on DHE should always be correct.

## Tested using browser:
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [x] Chrome (Desktop)

### CODE REVIEW

**Review by natalie-bernhard — DISMISSED** [review]

File: src/main/content/_assets/js/builds.js [hunk 1]
```diff
@@ -187,6 +188,46 @@ function startAnimation() {
...
+            }
+        }
+        if (package_names.includes("jakartaee10")) {
+            var j_index = package_names[i].indexOf("jakartaee9");
+            if (j_index > -1) {
+                builds[i].package_locations.splice(j_index, 1);
+                builds[i].package_signature_locations.splice(j_index, 1);
+            }
+        }
+        if (package_names.includes("webprofile10")) {
```
Comment (kinueng) [inline src/main/content/_assets/js/builds.js]:
```suggestion
        if (package_names.includes("webProfile10")) {
```

File: src/main/content/_assets/js/builds.js [hunk 2]
```diff
@@ -187,6 +188,46 @@ function startAnimation() {
...
+            var k_index = package_names[i].indexOf("webprofile9");
```
Comment (kinueng) [inline src/main/content/_assets/js/builds.js]: @natalie-bernhard I spotted* a mistake in my code. I was flip-floppping between making the code matching case insensitive. It turned out case sensitive matching is the better approach (as coded before this PR). 

My mistake is I left the `"webprofile*"` matches case insensitive.
```suggestion
            var k_index = package_names[i].indexOf("webProfile9");
```

**Comment by kinueng** [comment]
Hi @natalie-bernhard , please re-review this PR. See comment https://github.com/OpenLiberty/openliberty.io/pull/3114#discussion_r1148031456 for the re-review cause.

**Review by natalie-bernhard — APPROVED** [review]

**Comment by natalie-bernhard** [comment]
> Hi @natalie-bernhard , please re-review this PR. See comment [#3114 (comment)](https://github.com/OpenLiberty/openliberty.io/pull/3114#discussion_r1148031456) for the re-review cause.

Nice catch

**Pull request merged**